### PR TITLE
Modified CMakeLists.txt to include cpack and add variables for debian…

### PR DESCRIPTION
**Pull request by GalenRhodes**
Modified CMakeLists.txt to include cpack and add variables for debian packaging.

### CODE REVIEW

**Comment by nickhutchinson** [comment]
Hi, looks okay. How do I test this? I'm not familiar with cpack

**Comment by GalenRhodes** [comment]
I’ll get back to you in more detail on Sunday but the gist of it is after you run “make -j#” you then type “cpack -G DEB”.  The rest is pretty automatic.

--
Galen Rhodes
galenrhodes@me.com

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +15,21 @@ include (CheckCCompilerFlag)
 include (CheckCSourceCompiles)
 include (CMakePushCheckState)
 
+set(CPACK_STRIP_FILES					ON)
+set(CPACK_PACKAGE_CONTACT			 	"Not Me <noone@foobar.com>")
+set(CPACK_PACKAGE_NAME					"LIBDISPATCH-${DISPATCH_VERSION}")
```
Comment (nickhutchinson) [inline CMakeLists.txt]: any reason for the all-caps?

**Comment by nickhutchinson** [comment]
Thanks for introducing me to CPack -- I played with it for a bit and have committed bd1808980b04830cbbd79c959b8bc554085e38a1 to master inspired by this patch. I found with Ubuntu 14.04 I needed to lower the versions of the dependencies listed in the `CPACK_DEBIAN_PACKAGE_DEPENDS` requirements spec.

Does it look okay? If you need further changes feel free to update this PR.

**Comment by GalenRhodes** [comment]
It looks good.

Yes, the versions I picked for the depends were based on Ubuntu 15.10.  What I eventually want to do is to create a single umbrella package for installing clang, libdispatch, libobjc2 and GNUstep in one easy step. I’m especially doing this for ARM based boards like the Raspberry Pi and Odroid.    I have Debian packages (armv7 and aarch64) for everything except GNUstep.  That’s gonna be a beast…

--
Galen Rhodes
galenrhodes@me.com

**Comment by nickhutchinson** [comment]
good luck!

**Comment by nickhutchinson** [comment]
Addressed by bd18089

**Pull request closed without merging**